src/tabulate.js

```diff
@@ -4,8 +4,9 @@ import { th, tr, td, table, tbody, a, b, span, fragment } from "./html"
 export function tabulate(lcov, options) {
 	const head = tr(
 		th("File"),
+		th("Statements"),
 		th("Branches"),
-		th("Funcs"),
+		th("Functions"),
```

Comment:
I'd prefer to leave it as before with "Funcs". Less column width space used for what is otherwise 4 chars max: 100%

Reply:
I agree with @fungilation here, this should not be part of this PR.

Reply:
Changed as requested.